common-crafting.lic

```diff
@@ -12,26 +12,26 @@ module DRCC
     ['nothing in there'].include?(DRC.bput('look in cruc', 'nothing in there', 'you see'))
   end
 
-  def find_empty_crucible
+  def find_empty_crucible(idle_room = nil, crucibles)
```

Comment:
Actually I think I'll have the default argument lookup the hometown and it
can be overridden if in some case someone wanted to. Although I can't think
of a reason anyone would. See any issues with that?

On Tue, Sep 6, 2016 at 1:42 PM, Eric Meek <eric.meek71@gmail.com> wrote:

> Yes, I should have the lookups in the common- files. Changing that.
>
> On Tue, Sep 6, 2016 at 12:41 PM, Robert Herbig <notifications@github.com>
> wrote:
>
>> In common-crafting.lic
>> <https://github.com/rpherbig/dr-scripts/pull/751#discussion_r77672027>:
>>
>> > @@ -12,26 +12,26 @@ module DRCC
>> >      ['nothing in there'].include?(DRC.bput('look in cruc', 'nothing in there', 'you see'))
>> >    end
>> >
>> > -  def find_empty_crucible
>> > +  def find_empty_crucible(idle_room = nil, crucibles)
>>
>> Would it be any easier if you passed in a hometown string as an argument
>> and had these helpers do the lookup?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/rpherbig/dr-scripts/pull/751/files/d56664e67f7772e62511263960f200f4536e4317#r77672027>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/ARQKjsB6cksjymBOOAHd6bkF5WtGSi43ks5qnZeegaJpZM4J2CdL>
>> .
>>
>
>